loader/plugins/prune.js

```diff
@@ -0,0 +1,42 @@
+const CordlrPlugin = require('cordlr-plugin')
+
+module.exports = class HelpPlugin extends CordlrPlugin {
+  constructor (bot, config) {
+    super(bot, config)
+
+    this.name = 'Cordlr Core - Prune'
+    this.description = 'Cordlr Core plugin for pruning an specific amount of messages'
+
+    this.commands = {
+      'prune': {
+        'usage': '<x>',
+        'function': 'pruneMessages',
+        'description': 'Prunes the last x messages',
+        'permissions': [
+          'MANAGE_MESSAGES'

```

Comment:
It seems the bot is not able to delete messages from **Administrators**, I believe this is because administrator is a higher permission level. This results in many un-handled promise rejection messages in console.